gdc_client/exceptions.py

```diff
@@ -1,3 +1,6 @@
 class ClientError(Exception):
-    """ Base client error.
```

Comment:
I def like the idea here! Just have to say that these should prob be exceptions vs errors. Unless you are saying these are exceptions that we could never recover from.

Reply:
ClientError already existed, I just added the ServerError one.  The code right now suggests that if either of these errors occur, it'll basically end the program. If we wanted to change the name of ClientError, should be in another ticket, as it would touch other parts of the codebase.